test/analytics/tests/instrumented_events/from_the_browser/viewport_resize.ts

```diff
@@ -25,11 +23,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     });
 
     it('should emit a "viewport_resize" event when the browser is resized', async () => {
-      const events = await ebtUIHelper.getEvents(1, {
-        eventTypes: ['viewport_resize'],
-        withTimeoutMs: 100,
-      });
-      expect(events.length).to.be(0);
```

Comment:
This is the validation that failed. Probably coming from an unclean previous test.

It's not really relevant to this test, other than ensuring that we don't get old events later on. I've used `fromTimestamp` to achieve the same effect.